source/agora/node/Ledger.d

```diff
@@ -228,6 +228,9 @@ public class Ledger
         this.enroll_man.clearExpiredValidators(block.header.height);
 
         foreach (idx, ref enrollment; block.header.enrollments)
+        {
+            this.enroll_man.pool.remove(enrollment.utxo_key);
```

Comment:
https://github.com/bpfkorea/agora/blob/d2931dd0d0f552997bc80e799a5275f5c3cecff5/source/agora/consensus/EnrollmentManager.d#L214

Reply:
Ah, right. I think in my fixes for #836 I will remove the call within `addValidator` because that function can fail.